### PR TITLE
add template Bug Issue

### PR DESCRIPTION
**Pull request by alexsukhrin**

### CODE REVIEW

**Review by VShkaberda — COMMENTED** [review]
What about template for feature request? E.g., something like https://github.com/dbt-labs/dbt-spark/blob/main/.github/ISSUE_TEMPLATE/feature-request.yml

**Comment by alexsukhrin** [comment]
> What about template for feature request? E.g., something like https://github.com/dbt-labs/dbt-spark/blob/main/.github/ISSUE_TEMPLATE/feature-request.yml

looks good but will need to cook it right for us
maybe I'll make it a separate ticket

File: .github/ISSUE_TEMPLATE/bug-issue.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+* Mirroring library version:
+* Spark version:
+* Scala version:
+
+### Willingness to contribute
+
+The Delta Flow Community encourages bug fix contributions. Would you or another member of your organization be willing to contribute a fix for this bug to the Delta Flow code base?
+
+- [ ] Yes. I can contribute a fix for this bug independently.
+- [ ] Yes. I would be willing to contribute a fix for this bug with guidance from the Delta Lake community.
```
Comment (hanna-liashchuk) [inline .github/ISSUE_TEMPLATE/bug-issue.md]: Delta Lake?

Comment (alexsukhrin) [reply .github/ISSUE_TEMPLATE/bug-issue.md]: fix

**Review by hanna-liashchuk — APPROVED** [review]

**Review by VShkaberda — APPROVED** [review]

**Pull request merged**